File: pset6/sentimental-mario-more/mario.py
def main():
    while True:
        try:
            height = int(input("Height: "))
            if 1 <= height <= 8:
                break
            else:
                print("Please enter a number between 1 and 8.")
        except ValueError:
            print("Not a number.")

    for row in range(1, height + 1):
        space = height - row

        print(" " * space, end="")  # prints spaces for left side of pyramid
        print("#" * row, end="")  # prints hashes for left side of pyramid

        print("  ", end="")  # Print two spaces to separate the pyramids

        print("#" * row)  # prints hashes for right side of pyramid


if __name__ == "__main__":
    main()

File: pset1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int height = 0;

int main(void)
{

    // input, min: 1 and  max: 8
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    // loops through and create rows as long as the row is lower og equal to the height input
    for (int row = 1; row <= height; row++)
    {
        // left space of pyramid
        for (int space = height - row; space > 0; space--)
        {
            printf(" ");
        }

        // actual pyramid
        for (int hashSymbol = 1; hashSymbol <= row; hashSymbol++)
        {
            printf("#");
        }

        // gap between pyramids
        printf("  ");

        // right side of pyramid (without spaces)
        for (int hashSymbol = 1; hashSymbol <= row; hashSymbol++)
        {
            printf("#");
        }

        printf("\n");
    }
}